texas-hearing-institute/package.json

```diff
@@ -11,10 +11,18 @@
 		"prettier-format": "prettier . --write"
 	},
 	"dependencies": {
+    "@eva-design/eva": "^2.2.0",
+    "@eva-design/eva": "^2.2.0",
 		"@react-native-async-storage/async-storage": "^1.19.3",
```

Comment:
seems lint isn't working - can you look into it

Reply:
removed duplicates and linted